submissions/sprint_b/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[dev-packages]
+pytest = "*"
+
+[packages]
+flask = "*"
+python-dotenv = "*"
+
+[requires]
+python_version = "2.7"
```

Comment:
Python 2.7 is fine, but Flask recommends using the latest version of Python. I recommend Python 3.6
http://flask.pocoo.org/docs/1.0/installation/#installation